app/services/albums.js

```diff
@@ -0,0 +1,14 @@
+const request = require('request-promise'),
+  logger = require('../logger'),
+  errors = require('../errors');
+
+exports.getAlbumSources = url => {
+  const options = {
+    uri: url,
+    json: true
+  };
+  return request(options).catch(err => {
+    logger.error(err);
+    throw errors.albumsApiError(err.message);
```

Comment:
shouldnt this be serviceUnavailable error?

Reply:
Yes! I'm sorry, I was wrong. 👎